samples/sensor/lsm303dlhc/README.rst

```diff
@@ -0,0 +1,51 @@
+.. _lsm303dlhc:
+
+LSM303DLHC: Magnetometer and Accelerometer data Monitor
+#######################################################
+
+Overview
+********
+This sample application periodically reads magnetometer and accelerometer data
+from the LSM303DLHC, ST MEMS system-in-package featuring a 3D digital linear
+acceleration sensor and a 3D digital magnetic sensor and displays it on the
+console.
+
+Requirements
+************
+
+This sample uses the LSM303DLHC, ST MEMS system-in-package featuring a
+3D digital linear acceleration sensor and a 3D digital magnetic sensor,
+controlled using the I2C interface.
+
+References
+**********
+
+ - LSM303DLHC: http://www.st.com/en/mems-and-sensors/lsm303dlhc.html
```

Comment:
Since this is a one-item list, it would be better as a sentence:

    For more information about the LSM303DLHC eCompass module, see
    http://www.st.com/en/mems-and-sensors/lsm303dlhc.html